### DISCUSSION TITLE
Adding a second (third, fourth) encrypted dir to "base" dir does not work. Can't add files.

### DISCUSSION THREAD

**Discussion by pizzipie** [rfjakob/gocryptfs#726] [Q&A]
HI,
Using Dir _accounts_ to hold files for use in obtaining MySql Dbase info.
I created the _accounts_ encrypted Dir from scratch using **gocryptfs -init**. Mounted thusly:  
```
rick@rick-MS-7C02:~$ gocryptfs -allow_other ~/DBases/Dbmysql/acctsVault/acctsEncaccounts ~/DBases/Dbmysql/acctsVault/accounts
```
I can copy files to this Dir, store them,  and use **firefox  localhost/accts.com** to access my database etc.  All works fine.  
Directory structure is as follows: 
Dir: Dbmysql
    Dir: acctsVault
                              <----------- Want to add Dir: pizzidb 
                              <----------- Want to add Dir: pizEncpizzidb
       Dir: accounts
           Dirs & Files or Empty if not mounted
       Dir: acctsEncaccounts
           Encrypted Dirs & Files
           **Including gocryprtfs.conf**
       Dir: images
       File: accts.com.sh
       File: accts.txt
       File: index.html

Now I wanted to add another encrypted Dir to acctsVault so I did this:
```
 rick@rick-MS-7C02:~$ gocryptfs -allow_other ~/DBases/Dbmysql/acctsVault/pizEncpizzidb ~/DBases/Dbmysql/acctsVault/pizzidb. 
```
When this didn't work I added the **gocryptfs.conf** file from the  "acctsEncaccounts" Dir to the new "pizEncpizzidb" Dir. This mounted the "pizzidb" Dir ok. 
However when I try to add files to the "pizzidb" Dir I get the following: **Error opening file “/home/rick/DBases/Dbmysql/acctsVault pizzidb'_anyFileITryToAddHere_'”: No such file or directory**. 
Dir: Permissions:
drwxrwsr-x  2 rick www-data 4096 Apr  1 10:21 accounts/  &   pizzidb/
I also tried :
rick@rick-MS-7C02:~$ **~/DBases/Dbmysql/acctsVault/pizzidb$ touch testfile.txt
touch: cannot touch 'testfile.txt': No such file or directory**

Is there a proper way to do this or am I doing something wrong?

Thanks in advance for help on this.
R

**Comment by rfjakob** [comment]
Hi, didn't you miss `gocryptfs -init ~/DBases/Dbmysql/acctsVault/pizEncpizzidb` ?
